documentation/modules/ROOT/pages/connectors/oracle.adoc

```diff
@@ -1549,3 +1549,33 @@ Using the default `0` will mine all archive logs.
 When set to `0`, log mining history is is disabled.
 
 |===
+
+[[oracle-when-things-go-wrong]]
+== Behavior when things go wrong
+
+{prodname} is a distributed system that captures all changes in multiple upstream databases; it never misses or loses an event.
+When the system is operating normally or being managed carefully then {prodname} provides _exactly once_ delivery of every change event record.
+
+If a fault does happen then the system does not lose any events.
+However, while it is recovering from the fault, it might repeat some change events.
+In these abnormal situations, {prodname}, like Kafka, provides _at least once_ delivery of change events.
+
+The rest of this section describes how {prodname} handles various kinds of faults and problems.
+
+[id="oracle-cannot-reference-overflow-table"]
+=== ORA-25191 - Cannot reference overflow table of an index-organized table
+
+Oracle may issue this error during the snapshot phase when encountering an index-organized table (IOT).
+This error means that the connector has attempted to execute an operation that must be executed against the parent index-organized table that contains the specified overflow table.
+
+To resolve this, the IOT name used in the SQL operation should be replaced with the parent index-organized table name.
+To determine the parent index-organized table name, use the following SQL:
+
+```
+SELECT IOT_NAME
+  FROM DBA_TABLES
+ WHERE OWNER='<tablespace-owner>'
+   AND TABLE_NAME='<iot-table-name-that-failed>'
+```
+
+The connector's `table.include.list` or `table.exclude.list` configuration options should then be adjusted to explicitly include or exclude the appropriate tables to avoid the connector from attempting to monitor the child index-organized table.
```

Comment:
```suggestion
The connector's `table.include.list` or `table.exclude.list` configuration options should then be adjusted to explicitly include or exclude the appropriate tables to avoid the connector from attempting to capture changes from the child index-organized table.
```

Reply:
Let's use "monitor" only for monitoring (i.e. JMX), but not to refer to actually capturing changes.